scene/gui/tree.cpp

```diff
@@ -4150,7 +4220,7 @@ int Tree::get_column_minimum_width(int p_column) const {
 					depth += 1;
 				} else {
 					TreeItem *common_parent = item->get_parent();
-					while (common_parent != next->get_parent()) {
+					while (common_parent != next->get_parent() && common_parent) {
```

Comment:
Why this change? Doesn't seem related.

Reply:
This is to fix the issue that you raised near the top of this review.
Without this change your example code that didn't work will still not work.
I am not 100% sure why it is needed either, but adding this indeed fixes it.

Reply:
Ah, I see why is this needed now. I mean, I know when it could crash without this check, just not sure when this happens. Probably not important 🤔

You could swap the order though, so `common_parent` check is first.